### PR TITLE
Project Settings Metadata (fields and templates)

### PR DESCRIPTION
**Pull request by joshsadam**
## Description of changes

Returned metadata fields and templates to the settings panel.

## Related issue

N/A

## Checklist

Things for the developer to confirm they've done before the PR should be accepted:

*   [ ] ~CHANGELOG.md (and UPGRADING.md if necessary) updated with information for new change.~
*   [ ] ~Tests added (or description of how to test) for any new features.~
*   [ ] ~User documentation updated for UI or technical changes.~

### CODE REVIEW

File: src/main/webapp/resources/js/pages/projects/redux/fieldsSlice.js [hunk 1]
```diff
@@ -24,17 +33,23 @@ export const fetchFieldsForProject = createAsyncThunk(
...
   },
   reducers: {},
   extraReducers: {
-    [fetchFieldsForProject.fulfilled]: (state, action) => ({
-      ...state,
-      fields: action.payload,
-      loading: false,
-    }),
+    [fetchFieldsForProject.fulfilled]: (state, { meta, payload }) => {
+      const requests = { ...state.requests };
```
Comment (deepsidhu85) [inline src/main/webapp/resources/js/pages/projects/redux/fieldsSlice.js:43]: This is really awesome! Love how clean it is :)

File: src/main/webapp/resources/js/pages/projects/settings/index.js [hunk 2]
```diff
@@ -20,6 +26,9 @@ const ProjectMembers = React.lazy(() => import("./components/ProjectMembers"));
 const ProjectGroups = React.lazy(() =>
   import("./components/ProjectUserGroups")
 );
+// const MetadataFields = React.lazy(() =>
```
[the remark this answers is not in the document]
Comment (joshsadam) [reply src/main/webapp/resources/js/pages/projects/settings/index.js:29]: Sorry, removed in 99381adbd1f0f0aa48e010ccae478e8d43ea8b66

File: src/main/webapp/resources/js/pages/projects/redux/fieldsSlice.js (hunk 1 above)
Comment (joshsadam) [reply src/main/webapp/resources/js/pages/projects/redux/fieldsSlice.js:43]: Just wait until the next PR ;) So much simpler than this even.

Comment (deepsidhu85) [reply src/main/webapp/resources/js/pages/projects/redux/fieldsSlice.js:43]: Awesome!

**Review by deepsidhu85 — APPROVED** [review]

**Pull request merged**